### PR TITLE
Add Resource SCC V1 Project Notification Config

### PR DESCRIPTION
**Pull request by vijaykanthm**
Related to http://b/309603080.

Bug Description: https://github.com/hashicorp/terraform-provider-google/issues/16427

Adding new template for SCC API V1 Project Notification Config
https://cloud.google.com/security-command-center/docs/reference/rest/v1/projects.notificationConfigs

**Release Note Template for Downstream PRs (will be copied)**

```release-note:new-resource
`google_scc_project_notification_config`
```

### CODE REVIEW

**Review by c2thorn — COMMENTED** [review]
triggered build

**Review by c2thorn — CHANGES_REQUESTED** [review]
```
        Error: Unsupported argument
        
          on terraform_plugin_test.tf line 8, in resource "google_scc_project_notification_config" "custom_notification_config":
           8:   project_id   = ""
```

**Review by c2thorn — COMMENTED** [review]
running build

File: mmv1/products/securitycenter/ProjectNotificationConfig.yaml [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+  api: 'https://cloud.google.com/security-command-center/docs/reference/rest/v1/projects.notificationConfigs'
+examples:
+  - !ruby/object:Provider::Terraform::Examples
+    name: 'scc_project_notification_config_basic'
+    primary_resource_id: 'custom_notification_config'
+    vars:
+      topic_name: 'my-topic'
+      config_id: 'my-config'
+    test_env_vars:
+      project: :project
```
Comment (c2thorn) [inline mmv1/products/securitycenter/ProjectNotificationConfig.yaml:41]:
```suggestion
      project: :PROJECT_NAME
```

Comment (vijaykanthm) [reply mmv1/products/securitycenter/ProjectNotificationConfig.yaml]: addressed

**Review by c2thorn — CHANGES_REQUESTED** [review]
```
    vcr_utils.go:152: Step 2/2 error running import: ImportStateVerify attributes not equivalent. Difference is shown below. The - symbol indicates attributes missing after import.
        
        Â Â map[string]string{
        -Â 	"project": "<redacted id>",
        +Â 	"project": "<redacted num>",
        Â Â }
```
probably need to add `project` to `ignore_read_extra` in the example

File: mmv1/products/securitycenter/ProjectNotificationConfig.yaml [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+  api: 'https://cloud.google.com/security-command-center/docs/reference/rest/v1/projects.notificationConfigs'
+examples:
+  - !ruby/object:Provider::Terraform::Examples
+    name: 'scc_project_notification_config_basic'
+    primary_resource_id: 'custom_notification_config'
+    vars:
+      topic_name: 'my-topic'
+      config_id: 'my-config'
+    test_env_vars:
+      project: :PROJECT_NAME
```
Comment (c2thorn) [inline mmv1/products/securitycenter/ProjectNotificationConfig.yaml:41]:
```suggestion
    test_env_vars:
      project: :PROJECT_NAME
    ignore_read_extra:
      - 'project'
```

**Review by c2thorn — CHANGES_REQUESTED** [review]

File: mmv1/third_party/terraform/services/securitycenter/resource_scc_project_notification_config_test.go [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+		Steps: []resource.TestStep{
+			{
+				Config: testAccSecurityCenterProjectNotificationConfig_sccProjectNotificationConfigBasicExample(context),
+			},
+			{
+				ResourceName:            "google_scc_project_notification_config.custom_notification_config",
+				ImportState:             true,
+				ImportStateVerify:       true,
+				ImportStateVerifyIgnore: []string{"project", "config_id"},
+				IgnoreReadExtra: 		 []string{"project"},
```
Comment (c2thorn) [inline mmv1/third_party/terraform/services/securitycenter/resource_scc_project_notification_config_test.go:32]: My fix was not for this file, it already has `ImportStateVerifyIgnore: []string{"project"`, which is what `ignore_read_extra` does (for the generated file)

Comment (c2thorn) [reply mmv1/third_party/terraform/services/securitycenter/resource_scc_project_notification_config_test.go:32]: You need to remove these added lines, as they will cause the build to fail

Comment (vijaykanthm) [reply mmv1/third_party/terraform/services/securitycenter/resource_scc_project_notification_config_test.go]: Sure, will change it.

File: mmv1/products/securitycenter/ProjectNotificationConfig.yaml (hunk 2 above)
Comment (vijaykanthm) [reply mmv1/products/securitycenter/ProjectNotificationConfig.yaml:41]: updated

**Review by c2thorn — APPROVED** [review]

**Pull request merged**